modules/serverless-creating-a-kafka-event-sink.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * serverless/develop/serverless-event-sinks.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-creating-a-Kafka-event-sink_{context}"]
```

Comment:
Maybe use lower-case kafka in the id? - that way, it matches the file name

[id="serverless-creating-a-kafka-event-sink_{context}"]